gg.guilded.Guilded.yml

```diff
@@ -0,0 +1,57 @@
+app-id: gg.guilded.Guilded
+base: org.electronjs.Electron2.BaseApp
+base-version: "20.08"
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+sdk: org.freedesktop.Sdk
+command: guilded
+tags:
+  - proprietary
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --talk-name=org.freedesktop.Notifications
+  - --filesystem=xdg-videos:ro
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-download
+  - --talk-name=org.freedesktop.portal.Fcitx
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.indicator.application
+  - --talk-name=com.canonical.Unity.LauncherEntry
+  - --env=XDG_CURRENT_DESKTOP=ubuntu:GNOME
+  - --talk-name=org.freedesktop.secrets
+modules:
+  - name: guilded
+    buildsystem: simple
+    build-commands:
+      - ar x Guilded-Linux.deb data.tar.xz
+      - tar -xf data.tar.xz
+      - mv opt/Guilded /app/guilded
+      - sed -i 's|Icon=guilded|Icon=gg.guilded.Guilded|' usr/share/applications/guilded.desktop
+      - sed -i 's|Exec=/opt/Guilded/guilded|Exec=guilded|' usr/share/applications/guilded.desktop
+      - install -Dm644 usr/share/applications/guilded.desktop /app/share/applications/gg.guilded.Guilded.desktop
+      - install -Dm644 usr/share/icons/hicolor/16x16/apps/guilded.png /app/share/icons/hicolor/16x16/apps/gg.guilded.Guilded.png
+      - install -Dm644 usr/share/icons/hicolor/32x32/apps/guilded.png /app/share/icons/hicolor/32x32/apps/gg.guilded.Guilded.png
+      - install -Dm644 usr/share/icons/hicolor/48x48/apps/guilded.png /app/share/icons/hicolor/48x48/apps/gg.guilded.Guilded.png
+      - install -Dm644 usr/share/icons/hicolor/128x128/apps/guilded.png /app/share/icons/hicolor/128x128/apps/gg.guilded.Guilded.png
+      - install -Dm644 usr/share/icons/hicolor/256x256/apps/guilded.png /app/share/icons/hicolor/256x256/apps/gg.guilded.Guilded.png
+      - install -Dm644 usr/share/icons/hicolor/512x512/apps/guilded.png /app/share/icons/hicolor/512x512/apps/gg.guilded.Guilded.png
+      - install -Dm644 gg.guilded.Guilded.appdata.xml /app/share/appdata/gg.guilded.Guilded.appdata.xml
+      - install -Dm755 guilded.sh /app/bin/guilded
+    sources:
+      - type: file
+        url: https://www.guilded.gg/downloads/Guilded-Linux.deb
+        sha256: f9c64bae1ae6a478c42a87240e3734c82659f9f2a6a360f5f07769f0346040f7
+      - type: file
+        path: guilded.sh
+      - type: file
+        path: gg.guilded.Guilded.appdata.xml
+    cleanup:
+      - Guilded-Linux.deb
+      - data.tar.xz
+      - usr
```

Comment:
```suggestion
```
you don't need to cleanup build artifact.
cleanup is only for stuff that was installed by the install phase that you don't need.

Reply:
Sorry, I think I must of skimmed over that [in the docs](https://docs.flatpak.org/en/latest/manifests.html#cleanup). I've fixed this in https://github.com/flathub/flathub/pull/2470/commits/f933fd013b5052b813368d95ccdf62dfe0f7e377.